dcos/dcos-ui
# [spike] DCOS-21504: Switching from Service to Pod breaks the Networking tab

Check for containers in the pod networking tab
and if none exist, display a message.

Closes https://jira.mesosphere.com/browse/DCOS-21504

## Testing
1. Go to service tab and create a new service.
2. Choose single-container service.
3. In the first tab, click "Add another container" (on the bottom).
4. Open the Networking tab and verify that it works.
5. Verify that the added tests make sense.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![Peek 2019-05-16 13-18](https://user-images.githubusercontent.com/40791275/57850037-36a87c00-77dd-11e9-911f-b1ad5c4bfe65.gif)

### After
![Peek 2019-05-16 13-16](https://user-images.githubusercontent.com/40791275/57850043-3b6d3000-77dd-11e9-95d4-f613dacd4c4c.gif)
